index.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
         'jest/no-jasmine-globals': 'warn',
         'jest/no-test-prefixes': 'error',
         'jest/valid-describe': 'error',
-        'jest/valid-expect': 'error',
+        'jest/valid-expect': ['error', { alwaysAwait: true }],
```

Comment:
it should default to `false` to avoid a breaking change 🙂

Reply:
@SimenB After a second thought, since we are extending valid-expect capabilities, I think it would be more convenient to enable the options by default. Community have to upgrade their plugins to the minor version anyway, they will have this features enabled by default? Am I missing something?